### DISCUSSION TITLE
Deleted local project for repository

### DISCUSSION THREAD

**Discussion by Destaq** [community/community#23246] [New to GitHub]
<p>I accidentally permanently deleted the local version of my GitHub repository. This means that I can no longer push commits from my device. However, I would still like to edit my repository.</p>
<p>If I try to clone my repository, it doesn’t show as being the same repository as the one I have published. I do not want to delete and repost my repository as it already has a lot of stars, forks, etc. Is there any way that I can recreate a local version of my repository?</p>
<p>You can find my repo here: <a href="https://github.com/Destaq/chess_graph" rel="nofollow noopener">https://github.com/Destaq/chess_graph</a></p>

**Comment by airtower-luna** [comment, marked as answer]
<blockquote>
<p><a class="mention" href="/destaq">@destaq</a> wrote:</p>
</blockquote>
<blockquote>
<p>If I try to clone my repository, it doesn’t show as being the same repository as the one I have published.</p>
</blockquote>
<p>What do you mean by “doesn’t show as being the same”? Clone is should give your exactly what is available on Github.</p>
<p>To take a wild guess: Depending on how you authenticate for pushing you may need to set that up again for your new local repository.</p>
